### DISCUSSION TITLE
pip non reconnu

### DISCUSSION THREAD

**Discussion by mazid9912** [Nash115/frenchATC-for-MSFS2020#35] [Help] [answered]
lorsque j'installe les librairie il me met pip non reconnnu

**Comment by VyvaHart** [comment, marked as answer]
This issue is the most common when you start to use python.
From my point of view the best solution will be to reinstall python from main page: [Download Python](https://www.python.org/downloads/) (i recommend 3.11 version). And before installation mark the checkbox "Add to PATH"
after that open terminal (press WIN+R -> type 'cmd') and type python --version. If the terminal gives you: Python 3.11.0 - all good
then try to install/upgrade pip: python -m pip install --upgrade pip
i hope it will work for you

Good Luck!
